arches_vue_utils/src/arches_vue_utils/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface Language {
+    code: string;
+    default_direction: "ltr" | "rtl";
+    id: number;
+    isdefault: boolean;
+    name: string;
+    scope: string;
+}
+
+export interface Label {
+    value: string;
+    language_id: string;
+    valuetype_id: string;
+}
+
+export interface HasLabels {
+    labels: Label[];
+}
+
+export interface HasValues {
+    values: Label[];
+}
+
+export type Labellable = HasLabels | HasValues;
```

Comment:
does this abstraction need to exist?

Reply:
Good Q. In terms of how it's used, and maybe you grokked this already, it's so that we can clarify that the label finders work just as well on either type. If you're asking why I just don't write `HasLabels | HasValues` everywhere, I don't have a strong opinion. You can hover over to see the union anyway. I'm suggestible.

Reply:
yeah no worries, won't block on it 👍